### ISSUE TITLE
Manually add entries to the diary

### ISSUE THREAD

**Issue by MrReSc** [MrReSc/Headi#13]
A possibility to manually add entries to the diary would be helpful in some situations.
As a workaround, you can simply start a new recording and save it immediately. Afterwards the entry can be edited in the diary.

**Comment by wilayahapps** [comment]
I think its most needed

**Comment by MrReSc** [comment]
Yes, I know. However, I don't have much time for this project at the moment. 
As a workaround, you can simply start a new recording and save it immediately. Afterwards the entry can be edited in the diary.
